pkg/minishift/clusterup/clusterup.go

```diff
@@ -106,14 +107,8 @@ func ClusterUp(config *ClusterUpConfig, clusterUpParams map[string]string, runne
 
 // PostClusterUp runs the Minishift specific provisioning after 'cluster up' has run
 func PostClusterUp(clusterUpConfig *ClusterUpConfig, sshCommander provision.SSHCommander, addOnManager *manager.AddOnManager) error {
-	isOpenshift3_6 := util.VersionOrdinal(clusterUpConfig.OpenShiftVersion) >= util.VersionOrdinal("v3.6.0")
-
-	clusterName := getConfigClusterName(clusterUpConfig.Ip, clusterUpConfig.Port)
-	userName := fmt.Sprintf("system:admin/%s", clusterName)
-	// See issue https://github.com/minishift/minishift/issues/1011. Needs to be revisted whether needed with final OpensShift 3.6 (HF)
-	if isOpenshift3_6 {
-		userName = fmt.Sprintf("system:admin/127-0-0-1:%d", clusterUpConfig.Port)
-	}
+	// See issue https://github.com/minishift/minishift/issues/1011. Needs to be revisted if anything change for oc cluster up side
```

Comment:
@praveenkumar can you explain to me what's going on here? I think there are a few assumptions here which might be worth adding a comment about. 

How does this work? Pre 3.6 the username used contained the clustername/ip. In 3.6 it became 127-0-0-1. This is why we explicitly checked for >= 3.6 and if it was true we used 127-0-0-1. You removed this part now. The only way I can see this work is that if that if you are using oc >= 3.6 to provision a <= 1.5 version is not retrospectively uses  127-0-0-1 there as well. Is that correct? 

It also assumes that we use the default / baseline OpenShift version to provision versions of OpenShift which are <= this baseline. I recall that we discussed this in this form once.

I think we need to comment this somewhere. Not sure whether referencing #1011 provides the right context here.